### PR TITLE
use self-hosted runner

### PR DESCRIPTION
**Pull request by kumagi**
Github action is no more a free lunch?

### CODE REVIEW

File: .github/workflows/clang-format-validation.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 name: Validate code format with clang-format
 on: [push, pull_request]
+
 jobs:
   validate:
-    runs-on: ubuntu-latest
+    runs-on: self-hosted
```
Comment (twifkak) [inline .github/workflows/clang-format-validation.yml:6]: Maybe `[self-hosted, linux]` (per [doc](https://docs.github.com/en/actions/hosting-your-own-runners/using-self-hosted-runners-in-a-workflow#using-default-labels-to-route-jobs))?

**Review by twifkak — APPROVED** [review]
I don't think the latest actions failed. They were queued for 9 hours, e.g. https://github.com/google/nginx-sxg-module/actions/runs/642104064. This is surprising because https://github.com/google/webpackager/actions and https://github.com/ampproject/amppackager/actions don't show these delays. Maybe it was a temporary outage?

File: .github/workflows/clang-format-validation.yml (hunk 1 above)
Comment (kumagi) [reply .github/workflows/clang-format-validation.yml:6]: Just re-run resolved the failed CI. But I'll fix it. Thanks.

**Comment by twifkak** [comment]
Thanks. I'll merge as-is. No strong need to follow up with `[self-hosted, linux]`; it's only useful if we start self-hosting additional non-Linux runners. (e.g. Maybe in the future if runners are self-hosted at the org level instead of repo.)

**Pull request merged**